integrations/sbt-bloop/src/main/scala/bloop/integrations/sbt/SbtBloop.scala

```diff
@@ -195,6 +211,40 @@ object PluginImplementation {
       internalClasspath ++ externalClasspath
     }
 
+    def bloopCopyResourcesTask = Def.taskDyn {
+      val configKey = sbt.ConfigKey(Keys.configuration.value.name)
+      Def.task {
+        import sbt._
+        val t = AutoImported.bloopClassDirectory.value
+        val dirs =
+          Classpaths
+            .concatSettings(Keys.unmanagedResourceDirectories.in(configKey),
+                            AutoImported.bloopManagedResourceDirectories.in(configKey))
+            .value
+        val s = Keys.streams.value
+        val cacheStore = Compat.generateCacheFile(s, "copy-resources-bloop")
+        val mappings = (sbt.PathFinder(Keys.resources.value) --- dirs) pair (sbt.Path
+          .rebase(dirs, t) | sbt.Path.flat(t))
+        s.log.debug("Copy resource mappings: " + mappings.mkString("\n\t", "\n\t", ""))
+        sbt.Sync(cacheStore)(mappings)
+        mappings
+      }
+    }
```

Comment:
Can you add a comment explaining the difference between this task and sbt's implementation, and why we need to do this?

Reply:
The implementation is the same, but we need to hijack all the use sites of `targetDir`, and that means reimplementing this logic.
  
There's still work to do (I think, even though right now it works in our community build and all the projects i've tried it with), so I prefer to add the comment in a follow-up PR.